subhub/subhub_dynamodb.py

```diff
@@ -30,17 +30,39 @@ class Meta:
 class SubHubAccountModel(Model):
     user_id = UnicodeAttribute(hash_key=True)
     cust_id = UnicodeAttribute(null=True)
-    origin_system = UnicodeAttribute()
+    orig_system = UnicodeAttribute()
```

Comment:
we already corrected this from `orig_system` -> `origin_system`.  Let's not undo that work.

Reply:
this is still not addressed.